### PR TITLE
doc: Remove SHA-1 cipher suites from the defaults on the server-side

### PR DESCRIPTION
**Pull request by derekguo001**
Related PR: https://github.com/envoyproxy/envoy/pull/20643

Signed-off-by: derekguo001 <dong.guo@intel.com>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: api/envoy/extensions/transport_sockets/tls/v3/common.proto [hunk 1]
```diff
@@ -66,25 +66,17 @@ message TlsParameters {
   //
   //   [ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305]
   //   [ECDHE-RSA-AES128-GCM-SHA256|ECDHE-RSA-CHACHA20-POLY1305]
-  //   ECDHE-ECDSA-AES128-SHA
```
Comment (daixiang0) [inline api/envoy/extensions/transport_sockets/tls/v3/common.proto:69]: I am curious where we can check it is enabled by default, do we have tests for it?

Comment (derekguo001) [reply api/envoy/extensions/transport_sockets/tls/v3/common.proto:69]: Hi @daixiang0  There are test cases for the cipher suites in https://github.com/envoyproxy/envoy/blob/main/test/extensions/transport_sockets/tls/ssl_socket_test.cc

Comment (daixiang0) [reply api/envoy/extensions/transport_sockets/tls/v3/common.proto:69]: Good, lgtm.

**Review by htuch — APPROVED** [review]
Thanks! FWIW, @phlax it might be nice if we had a canonical list of default ciphers in one place that we consume for both code and docs, so we can ensure consistency here. Not super high priority, but one way to improve trust in doc accuracy.

**Pull request merged**